iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/util/ModificationReports.java

```diff
@@ -778,26 +778,29 @@ public static void generatorLocalRegulationReport(ReportNode reportNode, String
                 .add();
     }
 
-    public static void reportOnInconclusiveDryRun(ReportNode reportNode, String cause, String name) {
+    public static void reportOnInconclusiveDryRun(ReportNode reportNode, String cause, String modificationName, String networkName) {
         reportNode.newReportNode()
                 .withMessageTemplate("core.iidm.modification.networkModificationDryRun-failure")
                 .withUntypedValue("dryRunError", cause)
-                .withUntypedValue("networkModification", name)
+                .withUntypedValue("networkModification", modificationName)
+                .withUntypedValue("networkNameOrId", networkName)
                 .add();
     }
 
-    public static void dryRunReportNode(ReportNode reportNode) {
+    public static void dryRunReportNode(ReportNode reportNode, String modificationName, String networkName) {
         reportNode.newReportNode()
                 .withMessageTemplate("core.iidm.modification.networkModificationDryRun-success")
+                .withUntypedValue("networkModification", modificationName)
+                .withUntypedValue("networkNameOrId", networkName)
                 .withSeverity(TypedValue.INFO_SEVERITY)
                 .add();
     }
 
-    public static ReportNode reportOnDryRunStart(ReportNode reportNode, Network network, String name) {
+    public static ReportNode reportOnDryRunStart(ReportNode reportNode, String networkName, String modificationName) {
```

Comment:
For consistency with the other methods, can you invert `modificationName` and `networkName` arguments?